exporter/collector/config.go

```diff
@@ -100,6 +102,10 @@ type ResourceFilter struct {
 	Prefix string `mapstructure:"prefix"`
 }
 
+type LogConfig struct {
+	DefaultLogName string `mapstructure:"defaultLogName"`
```

Comment:
```suggestion
	DefaultLogName string `mapstructure:"default_log_name"`
```
config fields are separated by underscores instead of camelcase.

Reply:
👍